File: engine/src/main/java/com/jsoftware/platform/model/MonthlyPick.java
package com.jsoftware.platform.model;

import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Table(name="monthlypick")
public class MonthlyPick {
    private int id;
    private String title;
    private String img_url;
    private String contents;

    @Override
    public String toString() {
        return String.format("monthlypick[id='%d', title='%s', img_url='%s', contents='%s']", id, title, img_url, contents);
    }
}

File: flutter/lib/model/monthly_pick.dart

class MonthlyPick{
  final int id;
  final String title;
  final String contents;
  final String img_url;
  final String url;

  MonthlyPick({
    required this.id,
    required this.title,
    required this.contents,
    required this.img_url,
    required this.url,
  });

  //json 변환
  factory MonthlyPick.fromJson(Map<String, dynamic> json){
    return MonthlyPick(
      id: json['id'],
      title: json['title'] as String,
      contents: json['contents'] as String,
      img_url: json['img_url'] as String,
      url : json['url'] as String,
    );
  }
}